### DISCUSSION TITLE
Maybe a way for people to indicate how many dates they expect before "hooking up"

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#97] [Ideas & Features]
Different people take things at different speeds, and even if both parties are looking for a monogamous long term relationship, one party may want to wait months, and the other might want sex on the first date, which can lead to problems.

**Comment by Oshyan** [comment]
On OKCupid this is one of the many questions, e.g. "How many dates before you typically want to have sex?" or something, with options like 1, 3-5, 6 or more, I believe. Regardless of the specific options, I agree we should have this, and my only question then is whether it should be some dedicated feature, or more of just another question one could filter on like OKC does it. I lean toward the latter as time-to-intimacy is just one of many factors people might care a lot about, and I hope/intend that we'll have OKC-like questions to help create good matches.

**Comment by ajbott** [comment]
I believe an overarching dynamic filter feature can rule this feature out and it can be incorporated into people's searches in terms of what they are expecting in a relationship. If they are looking for a hookup, they can define if "sex before marriage" is something people value or "sex before date X" is something they value.
